docs/versioned-plugins/codecs/cloudtrail-v7.0.0.asciidoc

```diff
@@ -17,7 +17,7 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 === Cloudtrail codec plugin {version}
 
-// include::{include_path}/plugin_header-integration.asciidoc[]
```

Comment:
FYI:  Activating this file is what creates the short title to fix the right nav.  The header also pulls in info related to the integration, like indicating that this plugin is now part of an integration and linking to the integration repo and changelog. <img width="356" alt="Screen Shot 2022-07-25 at 5 13 32 PM" src="https://user-images.githubusercontent.com/35154725/181117274-82755522-0173-472b-a869-70b42f3d7d9d.png">